AutomatedTesting/Gem/PythonTests/EditorPythonTestTools/editor_python_test_tools/hydra_test_utils.py

```diff
@@ -89,13 +89,13 @@ def _check_for_listening_port(port):
         :return: True if port is listening.
         """
         port_listening = False
-        for conn in psutil.net_connections():
+        for conn in process_utils.psutil.net_connections():
```

Comment:
Fairly harmless, but still recommend continuing to call 3rdparty module psutil directly instead of through its import in process_utils

Reply:
Fixed!